backstop.json

```diff
@@ -16,2091 +16,1202 @@
       "height": 768
     }
   ],
+  "misMatchThreshold" : 0.1,
+  "readyEvent": "styleguideRendered",
   "scenarios": [
     {
       "label": "Button, generic - default",
       "url": "http://localhost:3000/section/1.1.1-1/fullscreen",
-      "hideSelectors": [],
-      "removeSelectors": [],
       "selectors": [
         ".object-section .o-button"
       ],
-      "readyEvent": "styleguideRendered",
-      "delay": 20,
-      "misMatchThreshold" : 0.1,
-      "onBeforeScript": "",
-      "onReadyScript": ""
+      "delay": 20
```

Comment:
I may well have missed the exception(s) as the list is mammoth, but it seems like every scenario uses a `delay: 20` — can we set that globally too? And if there are exceptions can we set it globally then override only for those?